src/tiledb/cloud/dag/dag.py

```diff
@@ -46,6 +49,8 @@
 from . import visualization as viz
 from .mode import Mode
 
+logger = logging.getLogger(__name__)
```

Comment:
For uniformity reasons, I prefer that we use `tiledb.cloud.utilities.get_logger_wrapper` for logging everywhere.

The method also allows for a verbosity level (`verbose=True/False`)  that should be set from the UDF's arguments (as is the case with the `as_batch` method)

Reply:
@JohnMoutafis I agree, but we have a circular import problem because `run_dag` is in the utilities module.

Reply:
@spencerseale @JohnMoutafis are we still stuck here? Do I understand correctly that we don't have a circular import yet, but *will* when `run_dag()` calls this new function?

If we refactored and moved `get_logger_wrapper()` to, for example, `tiledb.cloud.logging`, that would eliminate the potential circular import, yes? I'm willing to do that work.